tools/mfassignr/mfassignr_histnoise.xml

```diff
@@ -0,0 +1,44 @@
+<tool id="mfassignr_histnoise" name="MFAssignR HistNoise" version="@TOOL_VERSION@+galaxy0">
```

Comment:
please add a profile="" version

Reply:
```suggestion
<tool id="mfassignr_histnoise" name="MFAssignR HistNoise" version="@TOOL_VERSION@+galaxy0" profile="23.0">
```

Reply:
resolved in b8aa6958ebd2fa68c3f6dc0e15d0463e15f82c1e